featurebyte/utils/storage.py

```diff
@@ -98,6 +99,11 @@ def get_storage() -> Storage:
         return S3Storage(get_client=get_client, bucket_name=S3_BUCKET_NAME)
     if STORAGE_TYPE == "azure":
         return AzureBlobStorage(get_client=get_azure_storage_blob_client)
+    # TODO: This is a bad hack because we use a webdav settings in featurebyte-app
```

Comment:
I think we can omit this comment since the changes need to be done in different repo

Reply:
we changed to injected the storage variable instead